examples/notebooks/positive_Helmholtz_3D_VTK.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2443e734",
+   "metadata": {},
+   "source": [
+    "# 3D example with VTK and ParaView \n",
+    "\n",
+    "In this notebook we show how to solve a 3D problem in parallel, save the solution in HDF5 format and export it to VTK. \n",
+    "\n",
+    "We solve the positive Helmholtz equation in 3D with a callable function for the source. In practice, the source could be the interpolation of some experimental data. Besides, we impose periodic boundary conditions everywhere in the boundary.\n",
+    "\n",
+    "Let $\\Omega=[0, 10] \\times [-2\\pi, 2\\pi]\\times [-3, 1]$. We want to find $u\\in H(\\mathrm{curl}, \\Omega)$ such that\n",
+    "\n",
+    "$$ \\mathrm{curl} \\mathrm{curl} u + \\mu u = f, $$\n",
+    "\n",
+    "where $\\mu = \\pi$, and $f$ is a callable function in terms of the domain coordinates, the expression of which is a priori not known. \n",
```

Comment:
not very clear: if we don't know the expression of f, how can we evaluate its values ? 
I would write "$f$ is a callable function (of the domain coordinates) for which we may not have a symbolic expression"

Reply:
fixed in https://github.com/pyccel/psydac/pull/543/commits/8100732e415e617338e749e0ec061cbcaad91ca3